tests/Testdata/pos10.json

```
@@ -0,0 +1,11 @@
+{
+	"fullWidth": 6032,
```

Please indent the lines evenly.

```suggestion
  "fullWidth": 6032,
```